tests/test_main.py

```diff
@@ -1,11 +1,79 @@
+import pytest
 from fastapi.testclient import TestClient
+from src.main import project_CRUD_srvice
 
-from src.main import app
 
-client = TestClient(app)
+@pytest.fixture
+def client():
```

Comment:
Add return typing here, please. Keep in mind that we `yield` the value, not returning it